osu.Game/Localisation/NotificationsStrings.cs

```diff
@@ -113,6 +113,28 @@ public static class NotificationsStrings
         /// </summary>
         public static LocalisableString MismatchingBeatmapForReplay => new TranslatableString(getKey(@"mismatching_beatmap_for_replay"), @"Your local copy of the beatmap for this replay appears to be different than expected. You may need to update or re-download it.");
 
+        /// <summary>
+        /// "You are now running osu! {version}.
+        /// Click to see what's new!"
+        /// </summary>
+        public static LocalisableString GameVersionAfterUpdate(string version) => new TranslatableString(getKey(@"game_version_after_update"), @"You are now running osu! {0}.
```

Comment:
```suggestion
        public static LocalisableString GameVersionAfterUpdateToast(string version) => new TranslatableString(getKey(@"game_version_after_update_toast"), @"You are now running osu! {0}.
```

Reply:
Not really a toast (it's a notification and there are [`ToastStrings`](https://github.com/ppy/osu/blob/master/osu.Game/Localisation/ToastStrings.cs)), so no. I think what you had is fine as is.